albumentations/augmentations/transforms.py

```diff
@@ -106,17 +109,27 @@ class PadIfNeeded(DualTransform):
 
     def __init__(
         self,
-        min_height=1024,
-        min_width=1024,
+        min_height: Optional[int] = 1024,
+        min_width: Optional[int] = 1024,
+        pad_height_divisor: Optional[int] = None,
+        pad_width_divisor: Optional[int] = None,
         border_mode=cv2.BORDER_REFLECT_101,
         value=None,
         mask_value=None,
         always_apply=False,
         p=1.0,
     ):
+        if (min_height is None) == (pad_height_divisor is None):
+            raise ValueError("Only one of 'min_height' and 'pad_height_divisor' parameters must be set")
+
+        if (min_width is None) == (pad_width_divisor is None):
+            raise ValueError("Only one of 'min_width' and 'pad_width_divisor' parameters must be set")
+
```

Comment:
I think that better to compare boolean values using boolean functions
```python
if (min_height is None) and (pad_height_divisor is None):
    raise ValueError("Only one of 'min_height' and 'pad_height_divisor' parameters must be set")

if (min_width is None) and (pad_width_divisor is None):
    raise ValueError("Only one of 'min_width' and 'pad_width_divisor' parameters must be set")
```

Reply:
Ok, I understand that `==` works here like `xor` function, that is more correct.